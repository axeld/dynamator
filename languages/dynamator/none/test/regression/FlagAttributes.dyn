<dynamator language="none">
  <tag tag="input" name="t1">
    <attr name="checked">
      <discard/>
    </attr>
  </tag>
  <tag tag="input" name="t2">
    <attr name="checked">
      <content>checked</content>
    </attr>
  </tag>
</dynamator>
