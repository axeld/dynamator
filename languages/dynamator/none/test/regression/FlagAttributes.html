<html>
<head>
  <title>Test Flag Attributes</title>
</head>
<body>
<form>
<br>
<input type="checkbox" name="t1" checked>Should not be checked
<br>
<input type="checkbox" name="t2">Should be checked
</form>
</body>
</html>
