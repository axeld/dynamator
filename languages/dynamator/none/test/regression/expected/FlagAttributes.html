<html>
  <head>
    <title>Test Flag Attributes</title>
  </head>
  <body>
    <form><br>
      <input type="checkbox" name="t1">Should not be checked<br>
      <input type="checkbox" name="t2" checked>Should be checked</form>
  </body>
</html>
<!-- generated by Dynamator Fri Jan 30 21:32:55 CST 2004-->
